src/components/Board/Board.container.js

```diff
@@ -1614,7 +1614,12 @@ export class BoardContainer extends Component {
           onClose={this.handleTileEditorCancel}
           onEditSubmit={this.handleEditTileEditorSubmit}
           onAddSubmit={this.handleAddTileEditorSubmit}
-          boards={this.props.boards}
+          boards={this.props.boards.filter(
```

Comment:
this Looks incorrect to me. The boards included in the default cboard are not owned by the user who logged in.  You should not filter boards.

Reply:
If you dont filter, the select will show all the boards  in boards state. That contains the boards  created  before the user log in  and the boards created after the user log in. The filter takes only the user boards.
![image](https://user-images.githubusercontent.com/80465873/146981517-210dee3d-3b88-43ad-b19a-a6c5b69a25e1.png)
In the image you can see two home boards. You can check this issue, its a similar problem  #1102